packages/xstate-wallet/src/store/dexie-backend.ts

```diff
@@ -11,7 +11,11 @@ import {
 } from './types';
 import * as _ from 'lodash';
 
-import {Transaction, Dexie, TransactionMode} from 'dexie';
+// eslint-disable-next-line import/no-duplicates
+import Dexie from 'dexie';
```

Comment:
I think this can be written as `import Dexie, {Transaction, TransactionMode} from 'dexie';` so we don't have to ignore the eslint rules